readthedocs/oauth/services/base.py

```diff
@@ -34,6 +34,7 @@ class Service:
 
     adapter = None
     url_pattern = None
+    PROVIDER_SLUG = None
```

Comment:
Why don't we call it `VCS_PROVIDER` to match the field's name?

Reply:
@humitos This class is a VCS provider service class so It felt like I was referring to this class if I use ``VCS_PROVIDER``, so I used ``PROVIDER_SLUG``. but we can use ``VCS_PROVIDER_SLUG`` here. Or should I just use ``VCS_PROVIDER``?

Reply:
Either one is fine. It just called me the attention that we were using a different name here. Also, we can make it lowercase to match the pattern that we have with the other class variables used here as well.